### PR TITLE
Assign housing

### PR DESCRIPTION
**Pull request by adriannadiaz**
This only needs the front-end tests fixed

### CODE REVIEW

**Comment by adriannadiaz** [comment]
screen shot 2018-02-11 at 2 32 47 pm
Why do the items in the contacted resource array have an _id ?, the resource id is already as resource so this _id I don't know what it is for.

**Comment by adriannadiaz** [comment]
Passing frontend tests 
screen shot 2018-02-11 at 2 57 13 pm

**Comment by adriannadiaz** [comment]
Passing backend tests. 
screen shot 2018-02-11 at 2 58 29 pm

File: angular-client/src/app/components/participants/participant-profile/participant-profile.component.ts [hunk 1]
```diff
@@ -63,27 +75,52 @@ export class ParticipantProfileComponent implements OnInit {
...
+      this.loadParticipant();
     });
   }
 
+  loadCases(): void {
+    this.casefileService.getByParticipant(this.participantSelected._id)
+      .subscribe(data => {
+        if (data[0]) {
+          this.orderedCases = data as Array<Casefile>;
+          // TODO order cases by chrono order!
```
[the remark this answers is not in the document]
Comment (Ldhatter) [reply angular-client/src/app/components/participants/participant-profile/participant-profile.component.ts]: @adriannadiaz no this isnt the ordering of the search. this was a comment created a long time ago when we began work on casefiles.

@annarog  no the ordering isnt done yet. i can do that now

**Comment by annarog** [comment]
@adriannadiaz about the contacted resources _id:
Mongo adds it by default for any object. There's a way to disable this when creating a schema, but for array objects haven't found a way to remove it yet.

**Pull request merged**